gradio/utils.py

```diff
@@ -1090,7 +1083,7 @@ def is_in_or_equal(path_1: str | Path, path_2: str | Path):
         return ".." not in str(relative_path)
     except ValueError:
         return False
-    return True
+    return False
```

Comment:
When is this ever used? It seems like the function should terminate by lines 1083 or 105 at the latest

Reply:
Yes this is right - I think we can remove but just kept because I figured it was there for a reason lol